doc/source/_static/css/custom.css

```diff
@@ -245,8 +245,13 @@ th.tune-colab {
 
 /* Adjustment to Sphinx Book Theme */
 .table td {
-  /* Remove row spacing */
-  padding: 0;
+  /* Remove row spacing on the left */
+  padding-left: 0;
+}
+
+.table thead th {
+  /* Remove row spacing on the left */
+  padding-left: 0;
```

Comment:
This is required to make the summary page readable. Thanks @sijieamoy for suggesting this fix!

Before this change, the summary table looks bad as below (characters are packed together):

<img width="953" alt="Screen Shot 2022-12-20 at 11 59 13 AM" src="https://user-images.githubusercontent.com/4629931/208824547-99c3e7b8-b37f-4da4-a17f-6258c902e54f.png">